scripts/cloudlab/start_onenode_vhive_cluster.sh

```diff
@@ -36,16 +47,22 @@ echo Run the stock containerd daemon
 sudo containerd 1>$CTRDLOGDIR/ctrd.out 2>$CTRDLOGDIR/ctrd.err &
 sleep 1s
 
-echo Run the firecracker-containerd daemon
-sudo /usr/local/bin/firecracker-containerd --config /etc/firecracker-containerd/config.toml 1>$CTRDLOGDIR/fccd.out 2>$CTRDLOGDIR/fccd.err &
+if [ "$SANDBOX" == "gvisor" ]; then
+    echo Run the gvisor-containerd daemon
+    sudo /usr/local/bin/gvisor-containerd --address /run/gvisor-containerd/gvisor-containerd.sock --config /etc/gvisor-containerd/config.toml 1>$CTRDLOGDIR/gvisor.out 2>$CTRDLOGDIR/gvisor.err &
+elif [ "$SANDBOX" == "firecracker" ]; then
+    echo Run the firecracker-containerd daemon
+    sudo /usr/local/bin/firecracker-containerd --config /etc/firecracker-containerd/config.toml 1>$CTRDLOGDIR/fccd.out 2>$CTRDLOGDIR/fccd.err &
+fi
+
 sleep 1s
 
 echo Build vHive
 cd $ROOT
 source /etc/profile && go build
 
 echo Running vHive with \"${GITHUB_VHIVE_ARGS}\" arguments
-sudo ./vhive ${GITHUB_VHIVE_ARGS} 1>$CTRDLOGDIR/orch.out 2>$CTRDLOGDIR/orch.err &
+sudo ./vhive -sandbox ${SANDBOX} ${GITHUB_VHIVE_ARGS} 1>$CTRDLOGDIR/orch.out 2>$CTRDLOGDIR/orch.err &
 sleep 1s
 
 $SCRIPTS/cluster/create_one_node_cluster.sh
```

Comment:
Pass `gvisor` as an argument to `create_one_node_cluster.sh` and add a case for the same [here](https://github.com/nathanieltornow/vhive/blob/0725f4c140693b101d82a62caef0f21070a1c01d/scripts/cluster/create_one_node_cluster.sh#L33)

Reply:
I resolve it, I think that further modifications of `create_one_node_cluster.sh` are not required.